store/index.ts

```diff
@@ -396,7 +396,7 @@ export const useMainStore = defineStore('main', () => {
   ): Promise<Result<CryptoPayment, PaymentError>> => {
     try {
       const response = await fetchWithCsrf<CryptoPaymentResponse>(
-        '/webapi/payment/crypto',
+        '/webapi/payment/crypto/',
```

Comment:
Hmm does this call work on production?

something tells me this is one of the changes related to #241

Reply:
reverted